packages/driver/test/cypress/plugins/index.js

```diff
@@ -5,11 +5,17 @@ const _ = require('lodash')
 const path = require('path')
 const fs = require('fs-extra')
 const Promise = require('bluebird')
-const webpack = require('@cypress/webpack-preprocessor')
+const wp = require('@cypress/webpack-preprocessor')
 
 process.env.NO_LIVERELOAD = '1'
 const webpackOptions = require('@packages/runner/webpack.config.ts').default
 
+// set mode to development
```

Comment:
Can we change these comments to describe why these are being set instead of what they're being set to?

Reply:
done